config/validator.go

```diff
@@ -89,14 +89,30 @@ func (c *YamlConfig) RegionsEntryLogicalValidator() (valid bool, validationError
 	for regionName, regionConf := range c.Regions {
 		errList = append(errList, c.validateRegionCluster(regionName, regionConf)...)
 	}
-	if len(errList) > 0 {
-		valid = false
-		errorsList := make(map[string][]error)
-		errorsList["RegionsEntryLogicalValidator"] = errList
-		validationErrors = mergeErrors(validationErrors, errorsList)
+	validationErrors, valid = prepareErrors(errList, "RegionsEntryLogicalValidator")
+	return
+}
+
+// TransportsEntryLogicalValidator checks the correctness of "Transports" part of configuration file
+func (c *YamlConfig) TransportsEntryLogicalValidator() (valid bool, validationErrors map[string][]error) {
+	errList := make([]error, 0)
+	if len(c.Service.Client.Transports) == 0 {
+		errList = append(errList, errors.New("Empty transports definition"))
 	} else {
-		valid = true
+		lastTransportHasEmptyTriggers := false
+		for _, transportConf := range c.Service.Client.Transports {
+			methodIsDeclared, pathIsDeclared, queryIsDeclared :=
+				len(transportConf.Triggers.Method) > 0, len(transportConf.Triggers.Path) > 0, len(transportConf.Triggers.QueryParam) > 0
+
+			if !methodIsDeclared && !pathIsDeclared && !queryIsDeclared {
```

Comment:
```
if !methodIsDeclared && !pathIsDeclared && !queryIsDeclared {
   errList = append(errList, errors.New("No transport defined with empty \"Triggers\" in last item (dafault transport)"))
   break
} 
```
?

Reply:
Below suggestion doesn't work without additional condition.
Done (removing few lines of code).